src/components/tree-view/tree-node.st.css

```diff
@@ -3,23 +3,23 @@
     -st-named: color_Keyboard_Focused, color_Active;
 }
 
-.tree-node {
+.node {
```

Comment:
I don't get the subtlety of "node" vs. "item". Is there an actual difference? if not - I like `item`

Reply:
will make general clean up and improvements of naming across the component and its minions